app/controllers/application_controller.rb

```diff
@@ -52,7 +55,8 @@ def routing_error
   end
 
   def log_error(exception)
-    Raven.capture_exception(exception) if ENV['SENTRY_DSN'].present?
+    # report the original 'cause' of the exception when present
+    Raven.capture_exception(exception.cause.presence || exception) if ENV['SENTRY_DSN'].present?
```

Comment:
I don't think you need this whole cause thing. It only comes up if you catch an exception and raise a different type of exception as a result. Is that something we do?